cmd/neofs-node/config/grpc/config.go

```diff
@@ -0,0 +1,71 @@
+package grpcconfig
+
+import "github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+
+const (
+	subsection    = "grpc"
+	tlsSubsection = "tls"
+
+	ListenAddressDefault = "127.0.0.1:50501"
+	TLSCertFileDefault   = ""
+	TLSKeyFileDefault    = ""
+)
+
+type TLSConfig struct {
+	cfg *config.Config
+}
+
+// Endpoint returns value of "endpoint" config parameter
+// from "grpc" section.
+//
+// Returns ListenAddressDefault if value is not set.
+func Endpoint(c *config.Config) string {
+	v := config.StringSafe(c.Sub(subsection), "endpoint")
+	if v != "" {
+		return v
+	}
+
+	return ListenAddressDefault
+}
+
+// TLS returns structure that provides access to "tls" subsection of
+// "grpc" section.
+func TLS(c *config.Config) TLSConfig {
+	return TLSConfig{
+		cfg: c.Sub(subsection).Sub(tlsSubsection),
+	}
+}
+
+// Enabled returns value of "enabled" config parameter
```

Comment:
We don't need to specify parent sections here:
```suggestion
// Enabled returns value of "enabled" config parameter.
```

Reply:
fixed